test/passing/cmdline_override.ml.opts

```diff
@@ -0,0 +1 @@
+--config=module-item-spacing=compact --module-item-spacing=sparse
```

Comment:
Perhaps add a test with the order of options reversed to check that they are interpreted left-to-right

Reply:
Ok. I am just conscious of the potential for confusion. Maybe just make sure the doc of --config is clear that it is overridden by any other command line options, as I expect that the env var and --config being "the same" will only be anticipated by people who have implemented something with cmdliner.